docs/slicing.md

```diff
@@ -0,0 +1,145 @@
+# Slicing
+
+Functions (which includes strings, arrays, etc.) can be called using slice syntax, The syntax is as follows:
+
+```text
+someSet(optionalLowerBoundExpr;optionalUpperBoundExpr;optionalStepSizeExpr)
```

Comment:
Referring to upperBound, lowerBound and stepSize everywhere is overly verbose. Just use `start`, `stop` and `step` everywhere.